doc/cephfs/fs-nfs-exports.rst

```diff
@@ -120,7 +134,13 @@ serve this export.
 
 ``clusterid`` is the NFS Ganesha cluster ID.
 
-``binding`` is the pseudo root path (must be an absolute path).
+``binding`` is the pseudo root path (must be an absolute path and unique). It
+specifies the export position within the NFS v4 Pseudo Filesystem.
+
+``path`` is the path within cephfs. Valid path should be given and default path
+is '\'. It need not be unique. Subvolume path can be fetched using::
```

Comment:
```suggestion
is '/'. It need not be unique. Subvolume path can be fetched using::
```

Reply:
nit:

```suggestion
is '\'. It need not be unique. Subvolume path can be fetched using:

.. code::
```

Although this won't make any difference due to some config option in tool that converts RST files to HTML for Ceph docs, by default it does make a difference. Therefore, it's better to have it IMHO.